packages/gui/src/components/app/AppPassPrompt.tsx

```diff
@@ -65,6 +73,16 @@ export default function AppPassPrompt(props: Props) {
     if (e.key === 'Enter') {
       handleSubmit();
     }
+
+    if (e.getModifierState("CapsLock")) {
+      setShowCapsLock(true);
+    }
+  }
+
+  const handleKeyUp = (event) => {
```

Comment:
Suggestion for consistency with `handleKeyDown()`:

```suggestion
  function handleKeyUp(event: KeyboardEvent): void {
```
```